src/Config.php

```diff
@@ -1438,6 +1438,24 @@ function formatGlobalLayout(layout) {
             echo "</td></tr>";
         }
 
+        echo "<tr class='tab_bg_1'><th colspan='4' class='center b'>" . _n('Notification', 'Notifications', Session::getPluralNumber()) . "</th></tr>";
+
+        echo "<tr class='tab_bg_2'>";
+        $content_helper = "<ul>
+            <li>" . __('Preselect "notification: no" in ITIL objects actor configuration, with ability to derogate to it') . "</li>
```

Comment:
Not sure about it. Maybe @cconard96 could help.
```suggestion
            <li>" . __('Disable notifications by default on ITIL objects actor configuration, with ability to derogate to it') . "</li>
```